backend/src/test/java/heartbeat/service/source/github/GithubServiceTest.java

```diff
@@ -36,10 +33,7 @@
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.Mockito.when;
 
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
@weiraneve should not use `*` in any project, at least, our project, revert it

Reply:
done